wpeverest/everest-forms
# Fix - Ui-dragging

### All Submissions:






### Changes proposed in this Pull Request:



Fixes ui-dragging.

### How to test the changes in this Pull Request:

1. Drag a field in builder slowly and check if the dragging field hide under the builder or not.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Dragging field item being hidden.
